Link concepts "gRPC Ledger API" and "Ledger API" in glossary for #8029. (#8051)
digital-asset/daml: docs/source/concepts/glossary.rst

Ledger API
==========

The **Ledger API** is an API that's exposed by any `DAML ledger <#ledger-daml-ledger>`__.

> Alternative names: **DAML Ledger API** (formal name), **gRPC Ledger API** (informal name as it uses :doc:`/app-dev/grpc/index`).

Reviewer: ```suggestion
Alternative names: **DAML Ledger API** and **gRPC Ledger API** if disambiguation from other technologies is needed.
```

An idiomatic Scala library for writing `ledger applications <#application-ledger-client-integration>`__. See :doc:`/app-dev/bindings-scala/index`.

> gRPC Ledger API

Author: Because it does _not_ belong to the section "Ledger API libraries".
